### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by cxasper**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  create_table "users", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "email"
+    t.integer "mobile_phone"
+    t.integer "emergency_phone"
+    t.string "emergency_contact"
+    t.date "birthday"
+    t.boolean "is_admin"
```
Comment (lshimokawa) [inline db/schema.rb]: Remove is_admin, this should be a role.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# User
+class User < ApplicationRecord
```
Comment (lshimokawa) [inline app/models/user.rb]: Too many required fields, keep this at minimum.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'is valid with a firstname, lastname, email, mobilphone, birthday, is_admin, dni and address' do
+    user = User.new(
```
Comment (lshimokawa) [inline spec/models/user_spec.rb]: Use factory_bot gem

File: db/migrate/20180201000558_change_role.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class ChangeRole < ActiveRecord::Migration[5.1]
+  def change
+  	rename_column :users, :is_admin, :role
```
Comment (moisesnandres) [inline db/migrate/20180201000558_change_role.rb]: use 2 spaces instead of 4

File: spec/factories/user.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+
+FactoryBot.define do
+  factory :user do
+    first_name 'Example User'
+    last_name 'user@example.com'
+    email	'xxxx@email.com'
```
Comment (moisesnandres) [inline spec/factories/user.rb]: remove empty space

**Pull request closed without merging**